src/autopas/selectors/tuningStrategy/BayesianClusterSearch.h

```diff
@@ -164,6 +202,29 @@ class BayesianClusterSearch : public TuningStrategyInterface {
    */
   inline void sampleAcquisitions(size_t n, AcquisitionFunctionOption af);
 
+  /**
+   * FullSearch cannot handle continuous sets of cellSizeFactors.
+   * So we convert NumberSet to a finite std::set by sampling some values from the set.
```

Comment:
The first sentence is the justification that should probably be where the function is called and not its main documentation. The second sentence is the doc.
But shouldn't this be a function of NumberSet?